exp/etcdrestore/api/v1alpha1/etcdmachinesnapshot_types.go

```diff
@@ -46,6 +46,7 @@ type ETCDMachineSnapshotSpec struct {
 	ClusterName string `json:"clusterName"`
 	MachineName string `json:"machineName,omitempty"`
 	Location    string `json:"location,omitempty"`
+	Version     string `json:"version,omitempty"`
```

Comment:
Assuming this field is meant to be set by the system rather than a user, I suggest moving it to `ETCDMachineSnapshotStatus` instead.